### ISSUE TITLE
Log Spawn Points

### ISSUE THREAD

**Issue by Shanomac99** [Squad-Wiki/squad-wiki-pipeline-map-data#90]
Log player spawner points. Include the "Initial Life Spawn" element to show that some of them are temporary.

Discovered in #79.

**Comment by mahtoid** [comment]
How would you log the difference here in spawn points?

**Comment by Shanomac99** [comment]
@mahtoid Temporary spawn points/groups have an “initial life” which means that they expire after that initial life time has passed. Normal spawn points/groups have this set to 0.

I’m thinking I’m going to separate the spawn points into their spawn groups.

**Comment by sh4rkman** [comment]
Have a fix that i'll push in a PR once the current PR is closed : 

<img width="850" height="210" alt="Image" src="https://github.com/user-attachments/assets/7d636bdd-0f7c-4c55-a2c8-3acb2a5344ac" />

Data Extracted :

<img width="364" height="401" alt="Image" src="https://github.com/user-attachments/assets/085c86cb-113c-4960-9ff6-78205dfc7f99" />


Result in squadcalc : 

<img width="435" height="270" alt="Image" src="https://github.com/user-attachments/assets/10b36f9a-094c-4194-9678-1ef9e76f0ac0" />

**Comment by Shanomac99** [comment]
This looks good for spawn groups? IIRC we can also log the exact spots they can spawn at and associate it with a given spawn group.

**Comment by sh4rkman** [comment]
Yeah we can get every `SQTeamSpawnPoint` and log them, they seems all linked to a `spawnGroup` and have a `initialLifeSpan` which seems the same as their spawnGroup everytime.  

I honestly dont really care about them since having the `spawnGroup` looks enough to me but eh why not.  
Maybe @mahtoid is interested in them, or it can be helpfull on a specific gamemode ?

**Comment by Shanomac99** [comment]
I don't think he has specific spawn points like you have, figured it would be easy to grab and could be useful down the line to show specific spawn points in spawn.

**Comment by sh4rkman** [comment]
Alright i'll add it for my side for now and play with it
Do you want spawnGroups + spawnPoints in the current PR or the next one ?

**Comment by Shanomac99** [comment]
Lets push the current PR and then we can work on the next. If you want to check out https://github.com/Squad-Wiki/squad-wiki-pipeline-map-data/pull/125/commits/d05c9a392bee46e45c56e3287ee5bc3d0095a70a I think we are close to closing this PR.

**Comment by Shanomac99** [comment]
Fixed with #131
